undertow/src/main/java/org/wildfly/extension/undertow/UndertowRootDefinition.java

```diff
@@ -103,7 +102,7 @@
     protected static final SimpleAttributeDefinition DEFAULT_SECURITY_DOMAIN =
```

Comment:
Should we deprecate this attribute?

Reply:
I think we will be revisiting this attribute as part of another RFE so I don't think we should deprecate within these changes as it does still have an active meaning with an Elytron configuration.

Within the other features we are working on I think does introduce the question "Where should a default security domain be defined?"  IMO we have one subsystem that defines security domains so that should define which is the default instead of all of the other subsystems having their own notion of default.